packages/shared/sierra-client/src/HttpSierraClient.ts

```diff
@@ -6,13 +6,7 @@ import {
   successResponse,
   unhandledError,
 } from '@weco/identity-common';
-
-import {
-  extractRecordNumberFromLink,
-  PatronRecord,
-  toCreatePatron,
-  toPatronRecord,
-} from './patron';
+import { PatronRecord, toPatronRecord } from './patron';
```

Comment:
Why weren't these caught by linting previously?

Reply:
I don't know, I wondered the same!